net/unbound/files/unbound.init

```diff
@@ -16,10 +16,6 @@ PROG=/usr/sbin/unbound
 
 ##############################################################################
 
-. /usr/lib/unbound/unbound.sh
```

Comment:
Adding `$IPKG_INSTROOT` should fix the issue

Reply:
Thanks for the hint. Moving the source lines to function scope should also avoid the `rc.d` symlink process. The later seems less ugly because it is not checking for a make symbol in production.

Reply:
I think the symlink will happen anyway, regardless of where you put the source line.  Initscripts will be called with `enable` as the first argument and the symlink happens in the corresponding function inside `rc.common`.

Unless it is really expected to fail when sourcing unbound.sh on build machine, I am not aware of practice against including scripts at the top level.

Reply:
I double checked core init scripts and a few frequently updated packages. Sourcing in function scope only seems to be a common (only?) solution throughout OpenWrt. This includes `. /usr/lib/functions.sh` and others. I think I will stick with the precedent.

Reply:
Yes, that also works ;)